stable/spark/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+reviewers:
+- scottrigby
```

Comment:
@dbanda Just to be clear, this is not people who reviewed this PR, but people who want to maintain the chart moving forward. It looks like you want this to be:
```diff
approvers:
+ - dbanda
+ - mhamilton723
reviewers:
- - scottrigby
- - wirehead
+ - dbanda
+ - mhamilton723
```